core/src/test/scala/org/apache/spark/util/collection/unsafe/sort/RadixSortSuite.scala

```diff
@@ -157,7 +157,7 @@ class RadixSortSuite extends SparkFunSuite with Logging {
         buffer, N, sortType.startByteIdx, sortType.endByteIdx,
         sortType.descending, sortType.signed)
       val result = collectToArray(buffer, outOffset, N)
-      assert(ref.view == result.view)
+      assert(ref.sameElements(result))
```

Comment:
Ah, I faced this problem before in spark-xml. I believe you can use `===` instead, see also https://www.scalactic.org/user_guide/DefaultEquality. I remember it reports better when the arrays are not equal. e.g.)

```
scala.Predef.intArrayOps(data0).sameElements[Int](scala.Predef.wrapIntArray(scala.Array.apply(1, 2, 3))) was false
```

vs

```
Expected :Array(1, 2, 3)
Actual   :Array(-2147417843, -2147393996, -2146420562, -2146408050, -2145758434, ...
```

Reply:
Oh, let me try. Thanks

Reply:
Ur, do you mean to use `IndexSeqView` still like `ref.view === result.view`?

Reply:
nvm. I check that.